app/routes/application.js

```diff
@@ -2,11 +2,20 @@ import { action } from '@ember/object';
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 
+import * as Sentry from '@sentry/browser';
+
 export default class ApplicationRoute extends Route {
   @service progress;
+  @service router;
   @service session;
 
   beforeModel() {
+    this.router.on('routeDidChange', () => {
```

Comment:
Don't you want to put this in the router itself?

Reply:
hmm, I thought extending the router was discouraged these days 🤔

Reply:
Hhmm, yeah let's leave it like this.